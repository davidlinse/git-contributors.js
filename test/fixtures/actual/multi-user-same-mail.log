John Doe john@doe.org
 7 files changed, 7 insertions(+), 7 deletions(-)
John Doe john@doe.org
 1 file changed, 3 insertions(+), 4 deletions(-)
John Doe john@doe.org
 1 file changed, 6 insertions(+)
John Doe john@doe.org
 1 file changed, 6  deletions(-)
John Doe1 john@doe.com
 1 file changed, 6 insertions(+)
John Doe1 john@doe.com
 1 file changed, 6 insertions(+)
John Doe2 john@doe.com
 1 file changed, 6 insertions(+)
John Doe2 john@doe.com
 1 file changed, 6 insertions(+)
John Doe3 john@doe.com
 1 file changed, 6 insertions(+)
John Doe3 john@doe.com
 1 file changed, 6 insertions(+)

