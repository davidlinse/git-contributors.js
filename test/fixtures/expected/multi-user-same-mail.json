[
  {
    "commits": 4,
    "email": "john@doe.org",
    "name": "John Doe",
    "insertions": 16,
    "deletions": 17
  },
  {
    "commits": 2,
    "email": "john@doe.com",
    "name": "John Doe1",
    "insertions": 12,
    "deletions": 0
  },
  {
    "commits": 2,
    "email": "john@doe.com",
    "name": "John Doe2",
    "insertions": 12,
    "deletions": 0
  },
  {
    "commits": 2,
    "email": "john@doe.com",
    "name": "John Doe3",
    "insertions": 12,
    "deletions": 0
  }
]
